test/data.jl

```diff
@@ -54,7 +54,13 @@ module TestData
     @test size(df6, 2) == 3
 
     #test_group("null handling")
-    @test nrow(df5[complete_cases(df5), :]) == 3
+    @test nrow(df5[completecases(df5), :]) == 3
+    @test nrow(dropnull(df5)) == 3
+    returned_copy = dropnull(df4)
+    @test df4 == returned_copy && !(df4 === returned_copy)
+    @test nrow(dropnull!(df5)) == 3
+    returned_view = dropnull!(df4)
```

Comment:
What I mean is that a view is what `view()` returns, i.e. the parent isn't modified. Here, this is an in-place modification. Just a matter of terminology. (Anyway, you could use two the same name, we don't really care.)